neo/Consensus/ConsensusService.cs

```diff
@@ -114,8 +114,9 @@ private void CheckExpectedView(byte viewNumber)
             {
                 if (!context.WatchOnly())
                 {
-                    ChangeView message = context.ChangeViewPayloads[context.MyIndex]?.GetDeserializedMessage<ChangeView>();
-                    if (message is null || message.NewViewNumber < viewNumber)
+                    ChangeView myLastChangeViewMessage = context.ChangeViewPayloads[context.MyIndex]?.GetDeserializedMessage<ChangeView>();
+                    // Communicate the network about my agreement to move to `viewNumber`
+                    if (myLastChangeViewMessage is null || myLastChangeViewMessage.NewViewNumber < viewNumber)
```

Comment:
Please revert this variable rename Vitor... just add the comment `//Communicate the network...`

Reply:
The `message` name did not have any information, it was quite hard to guess the use of the variable.

Reply:
Reverted.